rtengine/iplocallab.cc

```diff
@@ -18000,8 +18006,15 @@ void ImProcFunctions::Lab_Local(
                         }
                     }
 
-
-
+                    bool execlaplace = true;//verify if others spot use dehaze
+                    /*
+                    const int sizespot = (int)params->locallab.spots.size();
+                    for (int sp = 0; sp < sizespot; sp++) {
+                        if(params->locallab.spots.at(sp).dehaz){//if true desable Laplace
+                            execlaplace = false;
+                        }
+                    }
+                    */
```

Comment:
It's ok to delete this code completely and remove the `execlaplace` variable.

Reply:
OK